docs-chef-io/content/server/upgrades.md

```diff
@@ -60,6 +60,12 @@ Chef Infra Server 14.14 supports external OpenSearch for indexing. Please follow
 1. Set the `opscode_erchef['search_auth_username']` attribute to OpenSearch username.
 1. Set the `opscode_erchef['search_auth_password']` attribute to OpenSearch password.
 
+{{< note >}}
+
+The OpenSearch user should have full access to the cluster: all cluster-wide operations, write to all indices. We recommend the backend role as admin `backend_roles:- "admin"`. Please refer the OpenSearch Roles for more details: https://opensearch.org/docs/latest/security-plugin/access-control/users-roles/#predefined-roles
```

Comment:
Would the admin user have the all_access role? I do not see admin user listed in the documentation provided.

Reply:
Yes Prajakta. Here is the link for the admin role details - https://opensearch.org/docs/latest/security-plugin/configuration/yaml#roles_mappingyml

Reply:
Ahh - I see, perhaps we can also include this link in the docs. 
Couple of questions:
- Could this note be after the example below?
- Is this opensearch user the same as the `opscode_erchef['search_auth_username']` that we would set on server?
- Could we clarify that we expect this user to be created by whoever is setting up external elasticsearch? (i.e reconfigure or any setup commands for server will not create this user on external opensearch setups)

Reply:
This user is same user as opscode_erchef['search_auth_username']. I will update the docs accordingly.
Thanks for the valuable feedback.

Reply:
Done